### ISSUE TITLE
PopOS detects two monitors when I only have one and I can't see the applications open because they open in the non-existent monitor.

### ISSUE THREAD

**Issue by All3xJ** [pop-os/pop#1246]
**Distribution (run `cat /etc/os-release`):**
```
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os
```
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
GRUB

**Issue/Bug Description:**
PopOS detects two monitors when I only have one and I can't see the applications open because they open in the non-existent monitor.

**Steps to reproduce (if you know):**
I have the issue either in Live PopOS USB and in installed PopOS. 

**Expected behavior:**
There should be only one monitor.

**Other Notes:**
I have this issue only in my Acer Aspire 5750G.

I found the solution here: https://askubuntu.com/a/365017

So basically you open the terminal and you run:

`xrandr | grep VGA`

and you note that output.

Now you run

`sudo gedit /etc/default/grub`

and at the line `GRUB_CMDLINE_LINUX="" `you have write:

`GRUB_CMDLINE_LINUX="video=VGA-1:d"`

But:

    If your output was VGA-1-1, set video=VGA-1:d (setting video=VGA-1-1:d won't work).

    If your output was VGA-1-2, set video=VGA-2:d (setting video=VGA-1-2:d or video=VGA-1:d
    does not seem to work, although one may still try them out separately to be certain).

Even though I fixed it, I just wanted to report it so you can fix it to help new Linux people.

**Comment by jaymcd** [comment]
Sadly I can't deploy this fix as I can't even get a terminal to open in Cosmic since I upgraded. May have to reinstall and then do this and then upgrade ? PITA.

**Comment by leviport** [comment]
This is a very old issue and is unrelated to COSMIC.

Detecting an extra display is often caused by the presence of a VGA port on the machine. Since it's an analog output that isn't negotiate with the host, the machine has no way to know whether something is actually plugged into the port or not, so it broadcasts that output constantly in case something is plugged in. If that's the cause, you probably have to just disable the extra display in settings and leave it that way.

Closing this issue due to age.

**Comment by jaymcd** [comment]
Why does it only happen on upgrade then ? 
  
  
 
  
 
 
 

-----Original Message-----

From: Levi ***@***.***>
To: pop-os ***@***.***>
Cc: jaymcd ***@***.***>; Comment ***@***.***>
Date: Wednesday, 29 April 2026 12:46 PM CDT
Subject: Re: [pop-os/pop] PopOS detects two monitors when I only have one and I can't see the applications open because they open in the non-existent monitor. (#1246)

 
leviport left a comment (pop-os/pop#1246) This is a very old issue and is unrelated to COSMIC. Detecting an extra display is often caused by the presence of a VGA port on the machine. Since it's an analog output that isn't negotiate with the host, the machine has no way to know whether something is actually plugged into the port or not, so it broadcasts that output constantly in case something is plugged in. If that's the cause, you probably have to just disable the extra display in settings and leave it that way. Closing this issue due to age. —
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by leviport** [comment]
I'm not sure what you're asking. If you're looking for technical assistance, consider joining Pop chat: https://chat.pop-os.org/pop-os/

This is an issue tracker, and isn't the best place to provide technical assistance.
